pyodide.asm.js is unavailable

### Bug Description

Hello, after updating to version 1.86.0 it is impossible to use Code nodes with Python.

Error is 

```json

{
  "errorMessage": "Cannot find module '/home/node/file:/usr/local/lib/node_modules/n8n/node_modules/src/js/pyodide.asm.js' imported from /usr/local/lib/node_modules/n8n/node_modules/pyodide/pyodide.mjs",
  "errorDetails": {},
  "n8nDetails": {
    "nodeName": "обрабатывает postgres",
    "nodeType": "n8n-nodes-base.code",
    "nodeVersion": 2,
    "n8nVersion": "1.86.0 (Self Hosted)",
    "binaryDataMode": "default",
    "stackTrace": [
      "Error: Cannot find module '/home/node/file:/usr/local/lib/node_modules/n8n/node_modules/src/js/pyodide.asm.js' imported from /usr/local/lib/node_modules/n8n/node_modules/pyodide/pyodide.mjs",
      "    at finalizeResolution (node:internal/modules/esm/resolve:283:11)",
      "    at moduleResolve (node:internal/modules/esm/resolve:952:10)",
      "    at defaultResolve (node:internal/modules/esm/resolve:1188:11)",
      "    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:542:12)",
      "    at ModuleLoader.resolve (node:internal/modules/esm/loader:510:25)",
      "    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38)",
      "    at ModuleLoader.import (node:internal/modules/esm/loader:472:34)",
      "    at defaultImportModuleDynamicallyForModule (node:internal/modules/esm/utils:214:31)",
      "    at url (node:internal/modules/esm/utils:253:12)",
      "    at loadScript (file:///usr/local/lib/node_modules/n8n/node_modules/src/js/compat.ts:219:69)",
      "    at loadPyodide (file:///usr/local/lib/node_modules/n8n/node_modules/src/js/pyodide.ts:308:11)",
      "    at processTicksAndRejections (node:internal/process/task_queues:95:5)",
      "    at LoadPyodide (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Code/Pyodide.js:38:23)",
      "    at PythonSandbox.runCodeInPython (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Code/PythonSandbox.js:58:21)",
      "    at PythonSandbox.runCodeAllItems (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Code/PythonSandbox.js:49:29)",
      "    at ExecuteContext.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Code/Code.node.js:155:17)"
    ]
  }
}

```

### To Reproduce

Just update to latest version

### Expected behavior

I'm expect thet pyodide will work properly

### Operating System

Ubuntu 22

### n8n Version

1.86.0

### Node.js Version

dont know, default

### Database

SQLite (default)

### Execution mode

main (default)

I'm experiencing this same issue and I installed via npm...

```
{
  "errorMessage": "Cannot find module '/home/n8n/file:/usr/local/lib/node_modules/n8n/node_modules/src/js/pyodide.asm.js' imported from /usr/local/lib/node_modules/n8n/node_modules/pyodide/pyodide.mjs",
  "errorDetails": {},
  "n8nDetails": {
    "nodeName": "code-get data",
    "nodeType": "n8n-nodes-base.code",
    "nodeVersion": 2,
    "n8nVersion": "1.86.0 (Self Hosted)",
    "binaryDataMode": "default",
    "stackTrace": [
      "Error: Cannot find module '/home/n8n/file:/usr/local/lib/node_modules/n8n/node_modules/src/js/pyodide.asm.js' imported from /usr/local/lib/node_modules/n8n/node_modules/pyodide/pyodide.mjs",
      "    at new NodeError (node:internal/errors:405:5)",
      "    at finalizeResolution (node:internal/modules/esm/resolve:327:11)",
      "    at moduleResolve (node:internal/modules/esm/resolve:980:10)",
      "    at defaultResolve (node:internal/modules/esm/resolve:1193:11)",
      "    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:403:12)",
      "    at ModuleLoader.resolve (node:internal/modules/esm/loader:372:25)",
      "    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:249:38)",
      "    at ModuleLoader.import (node:internal/modules/esm/loader:335:34)",
      "    at importModuleDynamically (node:internal/modules/esm/translators:143:35)",
      "    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:112:14)"
    ]
  }
} 
```

--- a/packages/nodes-base/nodes/Code/Pyodide.ts
+++ b/packages/nodes-base/nodes/Code/Pyodide.ts
@@ -1,3 +1,4 @@
+import { dirname } from 'node:path';
 import type { PyodideInterface } from 'pyodide';
 
 let pyodideInstance: PyodideInterface | undefined;
@@ -5,7 +6,8 @@ let pyodideInstance: PyodideInterface | undefined;
 export async function LoadPyodide(packageCacheDir: string): Promise<PyodideInterface> {
 	if (pyodideInstance === undefined) {
 		const { loadPyodide } = await import('pyodide');
-		pyodideInstance = await loadPyodide({ packageCacheDir });
+		const indexURL = dirname(require.resolve('pyodide'));
+		pyodideInstance = await loadPyodide({ indexURL, packageCacheDir });
 
 		await pyodideInstance.runPythonAsync(`
 from _pyodide_core import jsproxy_typedict

